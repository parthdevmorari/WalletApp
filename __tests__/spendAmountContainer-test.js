import 'react-native';
import React from 'react';
import SpendAmount from '../src/Components/SpendAmountContainer/spendAmount'
import Enzyme, {shallow, mount} from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import toJson from 'enzyme-to-json';

Enzyme.configure({adapter: new Adapter()});

describe('SpendAmount Component', () => {
  test('SpendAmount Component', () => {
    const wrapper = shallow(<SpendAmount />);
    expect(wrapper.exists()).toBe(true);
  });
  it('SpendAmount  toMatchSnapshot', () => {
    const wrapper = shallow(<SpendAmount />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});