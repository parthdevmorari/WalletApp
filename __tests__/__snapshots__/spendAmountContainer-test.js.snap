// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SpendAmount Component SpendAmount  toMatchSnapshot 1`] = `
<TouchableOpacity
  style={
    Object {
      "paddingTop": 60,
    }
  }
>
  <View
    style={
      Object {
        "alignSelf": "flex-start",
        "backgroundColor": "#e9fcf0",
        "borderRadius": 7.5,
        "paddingHorizontal": 52.5,
        "paddingVertical": 22.5,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#20D167",
          "fontSize": 22.5,
          "fontWeight": "bold",
          "textAlign": "center",
        }
      }
    >
      S$ 
    </Text>
  </View>
</TouchableOpacity>
`;

exports[`spendAmount Component spendAmount  toMatchSnapshot 1`] = `<spendAmount />`;
